app/controllers/api/openid_connect/clients_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   module OpenidConnect
     class ClientsController < ApplicationController
+      protect_from_forgery
```

Comment:
I feel `skip_before_action :verify_authenticity_token` to be more intelligible here, I would replace it.

Also I noticed there is one more place where it is required to suppress token verification. I want to build some simplified working web authentication client, before I update it, to make sure that I know all the changes required to make authentication working.